mix_tasks/update_english_translations.ex

```diff
@@ -1,8 +1,223 @@
 defmodule Mix.Tasks.UpdateEnglishTranslations do
   use Mix.Task
 
+  @cldr_url "https://raw.githubusercontent.com/unicode-org/cldr/main/common/subdivisions/en.xml"
+  @translations_dest Application.app_dir(:location, "/priv/iso_3166-2.en-translations.json")
+
   def run(_) do
-    Location.Country.load()
-    Location.Scraper.scrape()
+    {200, _headers, cldr_xml} = Location.HTTP.get!(@cldr_url)
+    cldr_translations = parse_cldr_xml(cldr_xml)
+
+    %{"3166-2" => debian} = read_json("priv/iso_3166-2.json")
+    restored = read_json("priv/restore/iso_3166-2.json")
```

Comment:
What's the purpose of the restore file?

Reply:
To remember the ISO codes that no longer exist in the real world but are still kept in Plausible database.

Reply:
Relevant PR where it was added: https://github.com/plausible/location/pull/17/files